src/layouts/DataImportBatch__c-NPSP Data Import Batch Layout.layout

```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We can't push upgrade changes to existing page layouts, so the steps to make these changes manually will need to be documented for customers who already have NPSP.

Reply:
I added a note about this to our docs outline for @smarovich.